SEP-0002-revised.md

```diff
@@ -0,0 +1,253 @@
+
+For the first election of the coordinating committee, one member will have a one year term, one member a two year term and one member a three year term.
+The term length of the elected candidates may be decided between themselves, in the case agreement can not be reached the person elected to the financial officer role will serve three years, the deputy lead developer two years and the lead developer one year.
+
+### Removing Steering Committee Members
+
+In exceptional circumstances, the project members may remove one or more sitting steering committee members via a single vote.
+
+Such a vote is triggered when a project member calls for one publicly on an appropriate Project-wide communication channel, and two other active project members second the proposal.
+
+For Steering Committee Member(s) to be removed, two-thirds of the active Project Members (where the count includes the member being voted on if they are a project member) must vote in support of the removal.
+
+To avoid repeated removal votes, any individual Project Member who has called for, or seconded such a vote, may not call for or second a vote to remove any Steering Committee member for one year from the original motion.
+
+## The Project Members
+
+The project members are trusted community members with a proven track record of constructive contribution to the goals of the project.
+Collectively, the project members make decisions about the direction of the project and have authority over the project, which is delegated to the Steering Committee.
+
+Project Members are expected to act as role models for the community and custodians of the SunPy Project, on behalf of the community and all those who rely on SunPy.
+They will act as representatives of the SunPy Project, where necessary, including in online discussions, in-person conferences, or at official SunPy events.
+
+### Responsibilities and Rights
+
+Project Members may and should participate in formal votes on:
+
+* Adding new Project Members
+* Electing Steering Committee members
+* Appointing the Ombudsperson
+* Removing Project Members
+* Removing Steering Committee members
+* Removing the Ombudsperson
+* Approving changes to this SEP via a vote (see below)
+* Other matters for which the Steering Committee believes a vote is appropriate.
+
+### Membership
+
+Project Members of the SunPy Project must demonstrate all of the following:
+
+* A good grasp of the philosophy of the SunPy Project
+* A solid track record of being constructive and helpful
+* Significant contributions to the Project's goals, in any form
+* A willingness to dedicate time to improving the Project
+* A willingness to recruit, train, and nominate new team members
+
+Project Membership acknowledges sustained and valuable efforts that align well with the philosophy and the goals of the SunPy Project.
+
+### Initial Membership
+
+To be able to elect new Project Members an initial set of Project Members are needed.
```

Comment:
As I spent much time arguing during the dev meeting today, I feel that we should be reviewing the initial set of invitees by name.  I think that trying to justify the initial set abstractly through seemingly objective criteria is silly.  These criteria aren't truly objective anyway, since entry into each of the listed pools was through a selection process that was subject to biases and judgements.

Reply:
I think explicitly listing people out is very preferential / in-club feeling, and I don't really like it. 

While you are obviously correct that these criteria give a list of people who have been "selected" by people in some way (mostly by me) they are also people who are *currently* active (with a guard for this with commit access).

While I don't like it, I could be persuaded to add the list of people these criteria select to an appendix of this document.

Reply:
> How about just using the author list of the SunPy paper as the initial membership group to be invited to be Project Members? We've been pretty open and transparent about that process.

Aside from the fact that that author list is stale, the majority of the people on that author list do not meet the criteria to be a Project Member

Reply:
We discussed this at the community today and we thought that including both the criteria and the list of names that evaluating that criteria results in, is the best compromise as we felt that having an unjustified list of names is equally arbitrary to an unresolved set of criteria.

Reply:
I'm okay with that compromise